### PR TITLE
 Included flood information in the HTML output

### PR DESCRIPTION
**Pull request by Solis13**
Adding some information of my Flood_APi into results.html @littlestripes @MAvecillas23

### CODE REVIEW

File: templates/results.html [hunk 1]
```diff
@@ -28,10 +28,24 @@ <h2>Air pollution in my city:</h2>
...
+        <li>Zone X: Low-risk flood zone</li>
+        <li>Other zones: Unknown flood risk</li>
+    </ul>
+
+    <p><strong>Flood Information:</strong> {{ Flood_info }}</p>  <!-- Display flood info -->
+
+    <p>
+        <strong>Flood Risk Level:</strong>
+        <span class="{% if flood_risk == 'safe' %}safe{% elif flood_risk == 'not safe' %}not-safe{% else %}unknown{% endif %}">
+            {{ flood_risk | upper }}
```
Comment (littlestripes) [inline templates/results.html:44]: `flood_risk` isn't passed in through `render_template` in `app.py`, so nothing is shown here and the correct CSS class isn't being applied. I'm also not sure what `upper` is, as this isn't defined either.

File: templates/results.html [hunk 2]
```diff
@@ -28,10 +28,24 @@ <h2>Air pollution in my city:</h2>
 
 <!--flood api results-->
 <div class="api-results">
-    <h2>Floods in my city:</h2>
-<p>{{Flood_info}}</p>
+    <h2>Flood Risk in My City</h2>
+    <p>Using FEMA's data, we assess the flood risk in your location:</p>
+    <ul>
+        <li>Zone A or Zone V: High-risk flood zones</li>
```
Comment (MAvecillas23) [inline templates/results.html:34]: most of the zone A, Zone V, Zone X stuff isn't needed considering that information won't be displayed to the user. To my understanding the only thing that will be returned from the flood module is either "safe" "unsafe" or unknown. Suggest specifying what 'Safe' 'Unsafe' 'Unkown' means instead.

**Comment by MAvecillas23** [comment]
all looks good. merging

File: templates/results.html [hunk 3]
```diff
@@ -46,14 +46,25 @@ <h2>How is my air quality?:</h2>
...
-        <h2>Floods in my city:</h2>
-        <p>{{Flood_info}}</p>
-    </div>
+<!--flood api results-->
+<div class="api-results">
+
+    <h2>Flood Risk in My City</h2>
+    <p>Using FEMA's data, we assess the flood risk in your location:</p>
+
+    <p><strong>Flood Information:</strong> <span class="{% if Flood_info == 'SAFE' %}safe{% elif Flood_info == 'NOT SAFE' %}not-safe{% else %}unknown{% endif %}">{{ Flood_info }}</span></p>  <!-- Display flood info -->
```
Comment (claraj) [inline templates/results.html:58]: Write this on multiple lines so it is easier to read

**Pull request merged**